source/manual/browser-support.html.md

```diff
@@ -5,8 +5,12 @@ layout: manual_layout
 section: Frontend
 type: learn
 owner_slack: "#govuk-frontenders"
-last_reviewed_on: 2018-08-29
+last_reviewed_on: 2019-10-08
 review_in: 12 months
 ---
 
-For browser support, see the Service Manual page about [Designing for different browsers and devices](https://www.gov.uk/service-manual/technology/designing-for-different-browsers-and-devices).
+GOV.UK [shares the same browser support matrix with GOV.UK Frontend](https://github.com/alphagov/govuk-frontend#browser-support) for pages served to the general public.
```

Comment:
GOV.UK Frontend supports IE 8 to a functional extent, whilst the Service Manual makes no mention of IE 8. This seems a bit confusing. I was under the impression that GOV.UK browser support was the same as listed in the Service Manual.

Reply:
The service manual defines the _minimum_ browser matrix that we ask services to test against. Some services may have different needs; for example, we _believe_ there are still services out there that have a higher percentage of traffic from IE8-10, and so they may choose to continue to support it. We want those services to be able to use GOV.UK Frontend, so we continue to support it at a functional level.